polkadot-protocol/architecture/system-chains/coretime.md

```diff
@@ -1,4 +1,59 @@
 ---
 title: Coretime
-description: TODO
----
\ No newline at end of file
+description: Learn about the role of the Coretime system parachain, which facilitates the sale, purchase, assignment, and mechanisms of bulk coretime.
+---
+
+## Introduction
+
+The Coretime system chain facilitates the allocation, procurement, sale, and scheduling of bulk [coretime](../../glossary.md#coretime), enabling tasks (such as [parachains](../../glossary.md#parachain)) to utilize the computation and security provided by Polkadot. 
+
+The [Broker pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=_blank}, along with [Cross Consensus Messaging (XCM)](todo:addlink), enables this functionality to be delegated to the system chain rather than the relay chain. Using XCMP's Upward Message Passing (UMP) to the relay chain allows for core assignments to take place for a task registered on the relay chain.
```

Comment:
Ideally, UMP should have a reference to the XCM's glossary within the XCM section. afaik that's not ready yet, but it would be nice to add it in the future

Otherwise, we can also link to - https://wiki.polkadot.network/docs/learn-xcm-transport#ump-upward-message-passing

```suggestion
The [Broker pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=_blank}, along with [Cross Consensus Messaging (XCM)](TODO:update-path){target=\_blank}, enables this functionality to be delegated to the system chain rather than the relay chain. Using [XCMP's Upward Message Passing (UMP)](TODO:update-path){target=\_blank} to the relay chain allows for core assignments to take place for a task registered on the relay chain.
```